MapboxNavigation/MGLStyle.swift

```diff
@@ -1,24 +1,26 @@
 import Foundation
 import Mapbox
 
+
 extension MGLStyle {
     // Returns the URL to the current version of the Mapbox Navigation Guidance Day style.
     @objc
-    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: "mapbox://styles/mapbox/navigation-guidance-day-v2")! } }
+    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: (MGLAccountManager.hasChinaBaseURL ? MGLAccountManager.mapboxChinaDayStyleURL : "mapbox://styles/mapbox/navigation-guidance-day-v2"))! } }
     
     // Returns the URL to the current version of the Mapbox Navigation Guidance Night style.
     @objc
-    public class var navigationGuidanceNightStyleURL: URL { get { return URL(string: "mapbox://styles/mapbox/navigation-guidance-night-v2")! } }
+    public class var navigationGuidanceNightStyleURL: URL { get { return URL(string: (MGLAccountManager.hasChinaBaseURL ? MGLAccountManager.mapboxChinaNightStyleURL : "mapbox://styles/mapbox/navigation-guidance-night-v2"))! } }
```

Comment:
For readability, put this code on multiple lines instead of using a ternary operator. (×4)